docs/content/tutorials/ransomware.md

```diff
@@ -0,0 +1,151 @@
+
+![Enable the SSH Exploiter](../../images/tutorials/hello-monkey/12-exploiter-enabled.jpg)
+
+#### Tell the Monkey which machine to target
+Make sure `hello` is provided as a target under _Scan target list_:
+
+![Scan target list in the Network Analysis configuration](../../images/tutorials/hello-monkey/5-scan-target-list.jpg)
+
+#### Tell the Monkey what credentials to use
+Enter `user` for _Identity_ and `password` for _Password_:
+
+![Credentials entered](../../images/tutorials/hello-monkey/13-credentials-input.jpg)
+
+#### Tell the Monkey to use the Ransomware plugin
+Next, we need to enable the Ransomware plugin. Select the **Payloads** tab, and enable the _Ransomware_ plugin in the _Enabled payloads_ list.
+
+![Ransomware enabled](../../images/tutorials/ransomware/2-ransomware-enabled.jpg)
+
+We also need to tell the Ransomware plugin which folder to hold for ransom. Select _Ransomware_ in the _Enabled payloads_ list in order to view its settings.
+
+Since the `hello` container is linux-based, and we want to target `/home/user/vault` folder, set the _Linux target directory_ to `/home/user/vault`.
+
+Also make sure that _Leave ransom note_ is checked.
+
+![Ransomware configured](../../images/tutorials/ransomware/3-ransomware-configuration.jpg)
+
+Don't forget to **Submit** the configuration!
+
+### Run Infection Monkey
+Now that we've configured Infection Monkey, let's run it!
+
+Go to the **Run Monkey** page and run the Monkey **From Island**.
+
+If you look at the **Infection Map** you'll see that the `hello` container gets exploited as it did in [Tutorial 1: Hello, Monkey](../hello-monkey).
+
+You can tell that the run has completed when a checkmark appears next to the **Infection Map** and **Security Report** in the navigation sidebar:
+
+![Infection Monkey run completed](../../images/tutorials/hello-monkey/7-run-monkey.jpg)
+
+
+Great, so the run has completed, but how do we know whether or not our ransomware succeeded?
+
+One good place to look is the **Ransomware report**. Navigate to the **Security Reports** page, and select the **Ransomware report** tab. Then take a look at the **3. Attack** section. It lists the files that Infection Monkey was able to encrypt:
+
+![List of files taken for ransom in ransomware report](../../images/tutorials/ransomware/4-ransomware-report.jpg)
+
+It looks like our passwords are being held for ransom! Let's connect to the container and observe the contents of the vault:
+```
+docker exec -it hello bash
+
+ls -1 /home/user/vault
+```
+
+We'll see two files:
+```
+README.txt
+passwords.txt.m0nk3y
+```
+
+`README.txt` is the ransom note that Infection Monkey leaves in the directory targeted by the Ransomware plugin. `passwords.txt.m0nk3y` is our password list (formerly `passwords.txt`) that Infection Monkey encrypted. You can print the contents of the file to verify that the file is indeed encrypted:
+
+```shell
+> cat /home/user/vault/passwords.txt.m0nk3y
```

Comment:
I think it's also worth instructing the user to `cat README.txt`, though I don't know that we need to necessarily display the output in the tutorial. Although:
![image](https://github.com/guardicore/monkey/assets/19957806/13faee59-0ffb-48fe-8d04-5c3efaa1e5dc)

Reply:
Maybe just link the file? They can crosscheck if they want to.